angular-spa-ui/js/search-app/search-app.js

```diff
@@ -1,24 +1,42 @@
 "use strict";
 
-function start() {
-    console.log('Search-app');
-    var searchApp = angular.module('searchApp', [require("angular-ui-router")]);
-    angular.bootstrap(document.getElementById("app"), ["searchApp"]);
+function app() {
+    console.log('App start');
+    var searchApp = angular.module('searchApp', ['ui.router']);
```

Comment:
Where you require ui.router?  I don't see any place where the ui.router was required into the current file